app/views/users/_create_form.html.erb

```diff
@@ -71,14 +77,16 @@
       %>
     </div>
 
-      <% if Rails.env == "production" && !params[:spamaway] %>
-        <div style="height: 120px">
-          <%= recaptcha_tags %>
-        </div>
+    <% if Rails.env == "production" && !params[:spamaway] %>
+      <div class="col-sm-12 recaptcha-container">
```

Comment:
HAHA fixed the Recaptcha issue -- it needed this class `col-sm-12`:

![image](https://user-images.githubusercontent.com/24359/52246448-e6ec1780-28b3-11e9-959b-0d490b47bdba.png)